erpnext/hr/doctype/employee/employee.json

```diff
@@ -843,12 +844,18 @@
    "fieldname": "twitter",
    "fieldtype": "Data",
    "label": "Twitter"
+  },
+  {
+   "fieldname": "email_id_of_reports_to",
+   "fieldtype": "Data",
+   "label": "Email ID of Reports to",
+   "options": "Email"
```

Comment:
It should be auto fetch when reports to employees added.

Reply:
there two records in employee doctype 1. company email 2. personal email
it should fetch company email, if company email is not present then it can fetch personal email

Reply:
This was not mentioned in Dod. I Will update accordingly and commit.